examples/rtlogger_helpers.py

```diff
@@ -63,44 +129,153 @@ def print_rtlogger_data(
 
     """
```

Comment:
Doesn't allow me to do a suggestion on lines other than the source diff, therefore as code snippet:
```python
def print_rtlogger_data(
    session: Session,
    awg: AWG,
    compensate_start_trigger: bool = True,
    max_lines: int = None,
    silent: bool = False,
) -> Optional[list[LogEntry]]:
    """Print the data collected by the RT Logger.

    Args:
        session: Toolkit session to a data server.
        awg: AWG node
        compensate_start_trigger: True if the start trigger delay should
            be compensated for.
        max_lines: Maximum number of lines to be printed.
        silent: if True - don't print anything, return the list of events. Default - False.

    Returns:
        list[LogEntry]: list of logged events, if silent is True, otherwise None

    """
```

Reply:
done